sbot/robot.py

```diff
@@ -33,26 +30,30 @@
     )
     ENABLE_VISION = False
 
-__version__ = "0.7.0"
+__version__ = "0.8.0dev"
 
 LOGGER = logging.getLogger(__name__)
 
 GAME_LENGTH = 120
 
 T = TypeVar("T", bound=Board)
+U = TypeVar("U", bound=Backend)
```

Comment:
I realise we have similar naming conventions in alot of place, but looking at this made we wonder if it'd be more useful to name bound TypeVars something like `TBoard` and `TBackend` instead of incrementing letters from `T`?

In this file there are ~70 lines between these declarations and their usages which is a way to scroll if you can't infer their meanings from other context clues.

Happy for this to be pulled out into a separate issue if you feel its out of the scope of this PR

Reply:
I agree. I don't think this is out of scope as I'm adding a new typevar anyway.

Reply:
1d80349